### ISSUE TITLE
Update to VS2017 + add NETSTANDARD 2.0 target

### ISSUE THREAD

**Issue by iamcarbon** [oocx/acme.net#18]
Are there plans to update the library to target the new csproj format / NETSTANDARD 2.0?

Am up for doing the port if this is a contribution that the project will accept.

- Jason

**Comment by oocx** [comment]
That would be a great contribution!

I'm currently working on another project in my free time, I won't have time to do the upgrade myself for at least the next several months.

**Comment by iamcarbon** [comment]
Awesome! getting started now.

**Comment by iamcarbon** [comment]
Ok, submitted a PR for NETSTANDARD 2.0 support. Going to work on extending the test coverage over the next few weeks.

**Comment by iamcarbon** [comment]
Just need to patch up the AppVeyor scripts and we should be GTG on the initial migration.

**Comment by oocx** [comment]
Great, thanks!

I just committed the appveyor.yml. I already changed the image to VS2017 and changed the targetframework in the project files from ".netcoreapp2.0" to ".netstandard2.0".

**Comment by iamcarbon** [comment]
Awesome. All looks well on the CI integration except for versioning.

Do we want to stick with the original format (0.0.0-beta-{build})) or should we begin managing these explicitly?

For instance: the next release could be 0.1.0-beta.

**Comment by oocx** [comment]
Yes, we should change the versioning to 0.1.0, the next release is a major change compared to the previous release that was based on a preview version of .net core.

I added you to the contributors list of the github project, so that you can checkin commits without me (we are in different time zones and I'm on vacation next week, and I don't want to slow you down).

**Comment by iamcarbon** [comment]
@oocx -- I think the code is in a good place for a 0.1.0-alpha release. 

Let me know if you have 10 minutes or so to chat on Skype with the longer term plans for the project. Am down to take a bigger role and help get this to v1. 

AM: jason AT carbonmade DOT com on Skype.

**Comment by iamcarbon** [comment]
@oocx Hope you had a good vacation! Let me know when you have a few to catch up and plan a release!

**Comment by oocx** [comment]
Thanks! Yes, the vacation was great.

It would be great if you wanted to work on this project. Personally, I don't have time to work on this project at the moment. So if you have ideas for improvements and refactorings, go ahead and implement them. When I created this repo, I just wanted to learn more about how Let's Encrypt work and did not actually plan to create and maintain a "real" project. I was surprised that it seemed to be useful to so many people. I'd be happy if you were willing to take over the project and move this project from a learning experiment / prototype to a real production ready tool :).
